guides/using-lastpass.md

```diff
@@ -4,50 +4,101 @@ expires: 2019-09-01
+
+### What it shouldn’t be used for
+
+LastPass should not be used for storing MOJ documents - you must use existing MOJ services such as Office 365 or Google G-Suite for that.
+
+You shouldn’t use LastPass for ‘secrets’ that belong to systems, only credentials to be used by humans. There is separate guidance on how to handle [secrets management](https://ministryofjustice.github.io/security-guidance/standards/secrets-management/).
+
+## How to use it
+
+### Getting started
+
+You will be sent an email to your MOJ work email account inviting you to create your LastPass account. LastPass have [‘getting started’ guides](https://support.logmeininc.com/lastpass?articleID=1194875481) on their website.
+
+#### Creating your master password
+
+You need to create a master password - this is the only password you'll need to remember.
+
+It must be at least 12 characters long (the longer the better).
+
+You can choose to make it pronounceable and memorable (passphrase) such as `CyberSecurityRules!` or `Sup3rD00p3rc0Mp3X!`, as long as you’re comfortable remembering it and won’t need to write it down.
+
+There are [password guidance standards the MOJ intranet](https://intranet.justice.gov.uk/guidance/security/it-computer-security/ict-security-policy-framework/password-standard/).
+
+Your master password **must** be unique and you should **never** use it anywhere else (including a similar version, for example, by simply adding numbers to the end)
+
+#### Multi-Factor Authentication
 
-## How to get it?
+You **must** setup multi-factor authentication (MFA, sometimes known as 2FA) for your MOJ LastPass account.
 
-(Ask Steve?)
+LastPass has a [guide on setting up MFA](https://support.logmeininc.com/lastpass/help/enable-multifactor-authentication-lp010002).
 
-## How to use Last Pass
+The MOJ has an ‘order of preference’ for [which types of MFA to use](https://ministryofjustice.github.io/security-guidance/standards/authentication/#multi-factor-authentication):
 
-### Getting Started
+* Hardware-based (for example, Yubikeys)
+* Software-based (for example, Google Prompt on a mobile device)
+* TOTP-based (the code is held by a dedicated app such as Google or LastPass Authenticator on a mobile device)
+* SMS-based (a one-time code sent via SMS)
 
-You will be sent an email inviting you to create your LastPass account. You need to create a master password - this is the only password you'll need to remember, but it is important that it's long and strong. It also has to be unique.
+If you don’t have an MOJ-issued work smartphone you may use a personal device for MFA.
 
-You next need to install the LastPass browser extension. ( How to do this? Self Service if you aren't on dev build?) 
+### Sharing passwords
 
-(something about only using work email address)
+To share a password [create a 'shared folder' in the LastPass Vault](https://support.logmeininc.com/lastpass/help/manage-lastpass-teams-shared-folders-users-lp010061)
```

Comment:
Something here about ensuring you only share with people who genuinely need to see the shared password / regularly reviewing this?